airsonic-main/src/main/java/org/airsonic/player/spring/SonosConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package org.airsonic.player.spring;
+
+import org.airsonic.player.service.SonosService;
+import org.airsonic.player.service.sonos.SonosFaultInterceptor;
+import org.airsonic.player.service.sonos.SonosHelper;
+import org.apache.cxf.jaxws.EndpointImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ImportResource;
+
+import javax.xml.ws.Endpoint;
+
+import java.util.Collections;
+
+@Configuration
+@ImportResource({"classpath:META-INF/cxf/cxf.xml", "classpath:META-INF/cxf/cxf-servlet.xml"})
+public class SonosConfiguration {
+
+    @Bean
+    public SonosFaultInterceptor sonosFaultInterceptor() {
+        return new SonosFaultInterceptor();
```

Comment:
These are ideal targets for autoinstantiation. Singleton? Contextual? No dependencies? Easy win.

Annotate the class and remove the bean from here.

Reply:
Done.